### PR TITLE
B2B-1260: Rebrand the android sdk to Mesh Connect

### PR DESCRIPTION
**Pull request by vitaliimovchan-front**

### CODE REVIEW

File: link/src/main/java/com/meshconnect/link/ui/LinkContract.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ *              Log.d("LinkResult", "Exited. ${result.errorMessage}")
+ *          }
+ *      }
+ *  }
+ *
+ *  linkButton.setOnClickListener {
+ *      linkLauncher.launch("linkToken")
+ *  }
+ * ```
+ */
```
Comment (AlexZeml-Front) [inline link/src/main/java/com/meshconnect/link/ui/LinkContract.kt:29]: do we need commented code ?

File: link/src/main/java/com/meshconnect/link/entity/AccessTokenPayload.kt [hunk 2]
```diff
@@ -15,7 +15,7 @@ data class AccessTokenPayload(
     val brokerBrandInfo: BrandInfo,
     val expiresInSeconds: Int? = null,
     val refreshTokenExpiresInSeconds: Int? = null,
-) : FrontPayload
+) : LinkPayload
 
 @Parcelize
 data class AccountToken(
```
Comment (AlexZeml-Front) [inline link/src/main/java/com/meshconnect/link/entity/AccessTokenPayload.kt:21]: As we decided to stick with Web SDK, I suggest to keep payload structure names and fields the same across platforms
BrokerAccountToken
BrokerAccount
BrokerBrandInfo
LinkPayload (need to rename FrontPayload in Web SDK)
AccessTokenPayload
DelayedAuthPayload

File: link/src/main/java/com/meshconnect/link/ui/LinkContract.kt (hunk 1 above)
Comment (vitaliimovchan-front) [reply link/src/main/java/com/meshconnect/link/ui/LinkContract.kt]: This code snippet is used for documentation. You can run `./gradlew link:dokkaHtml` to generate it.
https://imgur.com/a/D4aB3yg

Btw, this is a good catch, I forgot to update some lines. Thx.

File: link/src/main/java/com/meshconnect/link/entity/AccessTokenPayload.kt (hunk 2 above)
Comment (vitaliimovchan-front) [reply link/src/main/java/com/meshconnect/link/entity/AccessTokenPayload.kt:21]: There is a doc where we keep the names. I will dm you.

Comment (vitaliimovchan-front) [reply link/src/main/java/com/meshconnect/link/entity/AccessTokenPayload.kt:21]: @AlexZeml-Front added missing fields

**Review by AlexZeml-Front — APPROVED** [review]

**Review by afzalahmad-front — APPROVED** [review]
I took some time to go through it.  LGTM!

**Pull request merged**